scripts/accounts.ts

```diff
@@ -44,6 +44,8 @@ export function namedAccount(
   if (name == "l3sequencer") {
     return specialAccount(4);
   }
+  if (name == "espresso-sequencer")
+    return specialAccount(5);
```

Comment:
I wonder if the espresso sequencer account should be a user account instead? One could argue we don't require it to be a special account since it's we didn't increment [`specialAccounts`](https://github.com/EspressoSystems/nitro-testnode/pull/2/files#diff-0688a2ff615f28f7e2640a4e6edb7b5cd432cc038d537267c851bb3ea4ad26d3R8) but things are working anyway?

Reply:
Thanks! I didn't realize that. This constant is only to write down the information of accounts(for debugging?). Using `espresso-sequencer` as the name of the account maybe not necessary, but it clearly shows there is a special entity in our integration.